release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -931,6 +931,12 @@ For more information, see xref:../networking/hardware_networks/switching-bf2-nic
 
 Previously, for clusters that use the xref:../networking/ovn_kubernetes_network_provider/about-ovn-kubernetes.adoc#about-ovn-kubernetes[OVN-Kubernetes network plugin], during cluster installation you could enable hybrid networking so that your cluster supported Windows nodes. Now you can enable hybrid networking after installation. For more information, see xref:../networking/ovn_kubernetes_network_provider/configuring-hybrid-networking.adoc#configuring-hybrid-networking[Configuring hybrid networking].
 
+[id="ocp-4-12-networking-networkapi"]
+==== Support for allocateLoadBalancerNodePorts in Service object of Network API
+The `ServiceSpec` component in Network API under the `Service` object describes the attributes that a user creates on a service. The `allocateLoadBalancerNodePorts` attribute within the `ServiceSpec` component is now supported from {product-title} 4.12.28 release onwards. The `allocateLoadBalancerNodePorts` attribute defines whether the `NodePorts` will be automatically allocated for services of the `LoadBalancer` type.
```

Comment:
```suggestion
The `ServiceSpec` component in the Network API under the `Service` object describes the attributes that a user creates for a service. The `allocateLoadBalancerNodePorts` attribute within the `ServiceSpec` component is now supported as of {product-title} 4.12.28. The `allocateLoadBalancerNodePorts` attribute defines whether the `NodePorts` for `LoadBalancer` type services are automatically allocated.
```

Reply:
Thanks for the great comments, Agil!
For couple of suggestions above, I would keep it the same for consistency with the API docs [here](https://docs.openshift.com/container-platform/4.12/rest_api/network_apis/service-v1.html#spec).